INSTALL.md

```diff
@@ -263,4 +263,4 @@ make
 * opensuse-leap
 * ubuntu-22.04
 * alpine-3.15
-* FreeBSD
\ No newline at end of file
+* FreeBSD
```

Comment:
Maybe I'm stupid, but what is the change in this line ?!?

Reply:
A newline was either added or removed at the end of the file. It was on accident by me, I was using the GitHub web editor and it didn't show it as a change until after I opened my PR